ci/build_setup.sh

```diff
@@ -69,14 +69,14 @@ export BAZEL_TEST_OPTIONS="${BAZEL_BUILD_OPTIONS} --test_env=HOME --test_env=PYT
 [[ "${BAZEL_EXPUNGE}" == "1" ]] && "${BAZEL}" clean --expunge
 ln -sf /thirdparty "${ENVOY_SRCDIR}"/ci/prebuilt
 ln -sf /thirdparty_build "${ENVOY_SRCDIR}"/ci/prebuilt
-mkdir -p "${TEST_TMPDIR}/_bazel_${USER}/install"
-ln -sf /bazel-prebuilt-root/install/* "${TEST_TMPDIR}/_bazel_${USER}/install/"
 
 # Replace the existing Bazel output cache with a copy of the image's prebuilt deps.
-if [[ -d /bazel-prebuilt-output ]]; then
+if [[ -d /bazel-prebuilt-output && ! -d "${TEST_TMPDIR}/_bazel_${USER}" ]]; then
   BAZEL_OUTPUT_BASE="$(bazel info output_base)"
   rm -rf "${BAZEL_OUTPUT_BASE}"
```

Comment:
Not sure exactly what's going wrong with the old code, but if you want to avoid clobbering a fresher cache then this `rm -rf` probably needs to change also.

Reply:
It was symlinking in some stuff from the build image that Bazel complained about not being able to timestamp (?). Seems cleaner just to copy it rather than symlink to stuff generated by root on a read-only image.

The `rm -rf` is OK, but probably redundant, as we now have a check above to see if the cache exists; if it does, we don't bother trying to copy over it. We can probably actually just remove this line completely.